addons/knowledge/models/knowledge_article.py

```diff
@@ -645,26 +645,7 @@ def action_archive(self):
           * unreachable descendants (none, read) are set as free articles without
             root;
         """
-        all_descendants_sudo = self.sudo()._get_descendants()
```

Comment:
why not put it in the next commit then ?
(EDIT: ah because you use it in the two next commits)